### PR TITLE
config-refactor

### LINKED ISSUE #9: Bring base configuration used in the uds-package-software-factory into this repo
We want to move gitlabs configuration that is currently used at the [uds-package-software-factory](https://github.com/defenseunicorns/uds-package-software-factory) level to this repository.

We then want to provide a way for the capability to accept a values overlay.yaml file for additional configuration if necessary by the package that is importing this capability.

Acceptance Criteria:
- [ ] This gitlab runner capability now has a default configuration that mirrors what [uds-package-software-factory](https://github.com/defenseunicorns/uds-package-software-factory) is using.
- [ ] Makefile is updated to satisfy the dependencies it will need to for testing with the new configuration

### PR DESCRIPTION
**Pull request by Michael-Kruggel**
Closes #9

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -160,20 +167,30 @@ build/dubbd-pull-k3d.sha256: | build ## Download dubbd k3d oci package
...
 deploy/init: ## Deploy the zarf init package
 	./build/zarf init --confirm --components=git-server
 
 deploy/dubbd-k3d: ## Deploy the k3d flavor of DUBBD
 	cd ./build && ./zarf package deploy zarf-package-dubbd-k3d-amd64-$(DUBBD_K3D_VERSION).tar.zst --confirm
 
-deploy/uds-capability-gitlab-runner: ## Deploy the gilab capability
-	cd ./build && ./zarf package deploy zarf-package-gitlab-runner-*.tar.zst --confirm
+deploy/test-pkg-deps: ## Deploy the package dependencies needed for testing the gitlab capability
+	cd ./build && ./zarf package deploy zarf-package-swf-namespaces-* --confirm
```
Comment (anthonywendt) [inline Makefile:191]:
```suggestion
	cd ./build && ./zarf package deploy zarf-package-gitlab-runner-namespaces-* --confirm
```

File: utils/pkg-deps/namespaces/zarf.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/defenseunicorns/zarf/main/zarf.schema.json
+kind: ZarfPackageConfig
+metadata:
+  name: swf-namespaces
```
Comment (anthonywendt) [inline utils/pkg-deps/namespaces/zarf.yaml:4]:
```suggestion
  name: gitlab-runner-namespaces
```

**Comment by anthonywendt** [comment]
Can you see this?

**Comment by Michael-Kruggel** [comment]
> Can you see this?

No

**Review by anthonywendt — APPROVED** [review]
LGTM

**Pull request merged**